### PR TITLE
First upload to server

### PR DESCRIPTION
**Pull request by dbowiecag**
Finished the landing page and would like to put on server media.cag.org

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,14 +23,14 @@
 	<div id="menuWrapper">
 		<img id="menuLogo" src="img/cag-logo-left.png">
 		<div id="menuText">
-			 <a class="menu" id="requestButton">Requests</a>
-			 <a class="menu" id="serveButton"> Servant Ministers</a> 
+			 <a href="#requestForm" class="menu" id="requestButton">Staff Requests</a>
```
Comment (silentsonata) [inline index.html:26]: This dosn't look right

**Review by silentsonata — COMMENTED** [review]
Check out notes left

File: index.html (hunk 1 above)
Comment (dbowiecag) [reply index.html:26]: Nothing

**Review by silentsonata — APPROVED** [review]
Everything looks good!

**Pull request merged**